wdl-engine/tests/inputs/invalid-runtime-type/error.txt

```diff
@@ -0,0 +1,5 @@
+failed to parse input file `tests/inputs/invalid-runtime-type/inputs.json`
+
+Caused by:
+    0: invalid input key `test.runtime.returnCodes`
+    1: expected type `Int`, type `String`, or type `Array[Int]` for runtime key `returnCodes`, but found type `None`
```

Comment:
Do we want this to say `null` since that's what the value is in the JSON?

Reply:
I think it's best to consistently use WDL types in the error messages, as the expected types are always expressed as WDL types.

The [list of supported primitive types](https://github.com/openwdl/wdl/blob/wdl-1.2/SPEC.md#primitive-types-1) is pretty small and I don't think folks will get tripped up between WDL's `None` and JSON's `null`, to be honest.